ReText/highlighter.py

```diff
@@ -155,7 +157,37 @@ class ReTextHighlighter(QSyntaxHighlighter):
     )
 
     def highlightBlock(self, text):
-        # Syntax highlighter
+    # If inside fenced code block (not the fence line)
+        if self.previousBlockState() == 1:
```

Comment:
Please add the additional condition `self.docType == 'Markdown'` to this block and to the next one.